Finmer.Game/App.xaml.cs

```diff
@@ -55,6 +55,9 @@ protected override void OnStartup(StartupEventArgs e)
 
         protected override void OnExit(ExitEventArgs e)
         {
+            //Release Lua Context before Application is destroyed.
```

Comment:
This comment could describe in slightly more detail _why_ this needs to be done, not just _what_ is being done. For example, something like this:
```suggestion
            // Ensure all gameplay state is released while the app framework is still available
```